src/components/views/messages/RoomPredecessorTile.tsx

```diff
@@ -86,18 +88,32 @@ export const RoomPredecessorTile: React.FC<IProps> = ({ mxEvent, timestamp }) =>
     }
 
     const prevRoom = MatrixClientPeg.get().getRoom(predecessor.roomId);
-    if (!prevRoom) {
+
+    // We need either the previous room, or some servers to find it with.
+    // Otherwise, we must bail out here
+    if (!prevRoom && !predecessor.viaServers) {
         logger.warn(`Failed to find predecessor room with id ${predecessor.roomId}`);
-        return <></>;
-    }
-    const permalinkCreator = new RoomPermalinkCreator(prevRoom, predecessor.roomId);
-    permalinkCreator.load();
-    let predecessorPermalink: string;
-    if (predecessor.eventId) {
-        predecessorPermalink = permalinkCreator.forEvent(predecessor.eventId);
-    } else {
-        predecessorPermalink = permalinkCreator.forRoom();
+        return (
+            <EventTileBubble
+                className="mx_CreateEvent"
+                title={_t("This room is a continuation of another conversation.")}
+                timestamp={timestamp}
+            >
+                <div className="mx_EventTile_body">
+                    <span className="mx_EventTile_tileError">
+                        {_t("Can't find the old version of this room (room id: %(roomId)s).", {
```

Comment:
`Can't find the old version of this room` is accurate but it's a little confusing if you know the room exists.

Since we are even drilling down to the specific failure case of knowing that no via servers are present, we can probably use some language like: `Unable to find the old version of this room (!abc) because the upgrade info doesn't provide any servers to find it on.`

Ideally, we would even provide a way to ask the user if they would like to try to find the room via some server instead. But we can handle that in future iterations.

Reply:
I agree the best way would be to ask the user how they want to find this room, but that is outside the scope of this change.

As for the wording, it seems confusing to add more detail here. There's still nothing the user can do to fix the problem...

Reply:
An attempt is in eba4adbaf7 .

@richvdh - interested in your thoughts about whether this is bad form:

![image](https://user-images.githubusercontent.com/76812/229790922-28274009-1f49-41ba-adee-58e4ec4e4024.png)

I know in the past we have avoided parsing opaque IDs to prevent confusion. Here, it's part of an error message, and surrounded by caveats, so maybe we're alright with it? It gives the user a path that has a non-zero chance of success towards actually finding the thing they are looking for...

Reply:
yeah, I don't love it, though it's probably fine in this case.

It's worth noting that there is a plan to replace room ids with something properly opaque at some point. One day. Maybe.